src/components/signin/reducers.js

```diff
@@ -3,7 +3,7 @@ import * as TYPES from './actionType';
 const initialState = {
     errors: {},
     isAuthenticated: false,
-    loading: true,
+    loading: false,
     token: '',
```

Comment:
I'd advise you set the default token value to null. That way it is easier to check if token has a value